deps/spidershim/src/conversions.h

```diff
@@ -40,6 +40,10 @@ static inline Value* GetV8Value(JS::Value* val) {
   return reinterpret_cast<Value*>(val);
 }
 
+static inline Value* GetV8Value(JS::MutableHandleValue val) {
```

Comment:
Hmm, passing in a `MutableHandleValue` here looks wrong, since that type is intended only for out arguments...  As far as I can tell, `GCVector::operator[]` returns a `const JS::Value&' so I'm not sure why you need this overload.